Gemfile.lock

```diff
@@ -20,10 +25,11 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  pry
   rspec
   rspec-core
   rspec-expectations
   rspec-mocks
 
 BUNDLED WITH
-   1.16.1
+   2.6.6
```

Comment:
I bumped bundler as it was actually raising an error due to the newer version of Ruby that was being used. I can't see any harm in bumping it.